lib/contentTypeParser.js

```diff
@@ -149,7 +149,11 @@ ContentTypeParser.prototype.run = function (contentType, handler, request, reply
   const resource = new AsyncResource('content-type-parser:run', request)
 
   if (parser === undefined) {
-    reply.send(new FST_ERR_CTP_INVALID_MEDIA_TYPE(contentType || undefined))
+    if (request.is404) {
+      return handler(request, reply)
```

Comment:
just to avoid returning something 

```suggestion
     handler(request, reply)
     return
```